README.md

```diff
@@ -1,6 +1,6 @@
 # Portfolio-Setup
 
-## Portfolio Setup Project
+## Portfolio Setup
 
 This project is a replicate of figma mobile template one
 
```

Comment:
[Optional] For some reason I cannot make an inline comment directly on your `Getting started` section below, for that reason, permit me make the comment up here. Please kindly provide the URL to be cloned as you don't expect a user to go look for it. This will also add the professionalism of your README.

`git clone git@github.com:ismailakinkunmi/portfolio-site.git`
`cd portfolio-site`